sdk/core/core-xml/package.json

```diff
@@ -18,37 +18,29 @@
-    "build:test:browser": "npm run build:ts && npm run bundle:test:browser",
-    "build:test:node": "npm run build:ts && npm run bundle:test:node",
-    "build:ts": "tsc -p .",
+    "build:samples": "echo Obsolete",
+    "build:test": "tsc -p . && rollup -c 2>&1",
     "build:types": "downlevel-dts types/latest/ types/3.1/",
-    "build": "npm run build:ts && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
-    "bundle:test": "rollup -c rollup.test.config.js 2>&1",
-    "bundle:test:browser": "cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
-    "bundle:test:node": "cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "check-format": "prettier --list-different \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src",
     "execute:samples": "echo skipped",
-    "extract-api": "npm run build:ts && api-extractor run --local",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json api-extractor.json src test --ext .ts",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test:browser && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test:node && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run clean && npm run build:ts && npm run bundle:test:node && npm run unit-test:node && npm run bundle:test:browser && npm run unit-test:browser && npm run integration-test:node && npm run integration-test:browser",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run clean && tsc -p . && npm run unit-test:node && rollup -c 2>&1 && npm run unit-test:browser && npm run integration-test",
```

Comment:
Can you simplify this with build:test?

Reply:
This was needed to address @xirzec's comment in https://github.com/Azure/azure-sdk-for-js/pull/16248#discussion_r665688597. I am also reluctant to create `build:test:node` and `build:test:browser` so we do not bloat our npm scripts list for no good reason.